neon.yaml

```diff
@@ -0,0 +1,68 @@
+package:
+  name: neon
+  version: 3819
+  epoch: 0
+  description: "Serverless Postgres. We separated storage and compute to offer autoscaling, branching, and bottomless storage."
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - rust
+      - openssl-dev
+      - build-base
+      - libtool
+      - readline-dev
+      - zlib-dev
+      - flex
+      - bison
+      - libseccomp-dev
+      - clang
+      - pkgconf
+      - libpq-15
+      - cmake
+      - postgresql-15-client
+      - protoc
+      - libcurl-openssl4
+      - poetry
+      - perl
+      - bash
+      - coreutils
+      - protobuf
+      - curl-dev
+      - protobuf-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/neondatabase/neon
+      tag: release-${{package.version}}
+      expected-commit: 1804111a020390ac83d3517e5a4ef3c15f1656a7
+
+  - runs: |
+      git submodule update --init --recursive
+
+      make -j$(nproc) -s BUILD_TYPE=release
+
+      mkdir -p "${{targets.destdir}}"/usr/bin
+      for f in pg_sni_router pageserver pagectl safekeeper storage_broker proxy neon_local; do
+        mv target/release/$f "${{targets.destdir}}"/usr/bin
+      done
+
+      mkdir -p "${{targets.destdir}}"/usr/local
+      mv pg_install/v14 "${{targets.destdir}}"/usr/local/v14
```

Comment:
I don’t think we should install files into /usr/local.  Can we patch in a different install location?

Reply:
Yeah that part is simple, it's just for the final container image so I think we can symlink it via a compat sub package.